index.js

```diff
@@ -270,16 +269,11 @@ module.exports = function(S) {
               let destPath = path.join(_this.optimizedDistPath, p);
 
               if (fs.lstatSync(p).isDirectory()) {
-                wrench.mkdirSyncRecursive(destPath, '0777');
-                wrench.copyDirSyncRecursive(
-                  path.join(_this.evt.options.pathDist, p),
-                  destPath, {
-                    forceDelete: true
-                  }
-                );
+                fs.mkdirsSync(destPath, '0777');
+                fs.copySync(path.join(_this.evt.options.pathDist, p), destPath, {clobber: true, dereference: true});
               } else {
-                wrench.mkdirSyncRecursive(path.dirname(destPath), '0777');
-                S.utils.writeFileSync(destPath, S.utils.readFileSync(p));
+                fs.mkdirsSync(path.dirname(destPath), '0777');
+                fs.copySync(p, destPath, {clobber: true, dereference: true});
```

Comment:
According to the link I posted in the other task, the `dereference` option is only supported for the `fs-extra.copy` and not the `fs-extra.copySync`. I would suggest to use the asynchronous variant here, maybe wrapped by a Promise.fromCallback() to have a promisified implementation.
See also here: https://github.com/jprichardson/node-fs-extra/issues/210